### PR TITLE
Add release notes 0.4.6

### PR DESCRIPTION
**Pull request by sofroniewn**
Adds release notes for 0.4.6

### CODE REVIEW

File: docs/release/release_0_4_6.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+## Highlights
+This release is the first that adds support for persistant settings in napari (#2212).
```
Comment (tlambert03) [inline docs/release/release_0_4_6.md]:
```suggestion
This release is the first that adds support for persistent settings in napari (#2212).
```

File: docs/release/release_0_4_6.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+Right now we just store the current theme and window geometry but we will be expanding 
+this to include a full set of preferences in future releases.
+
+We've also made plugin installation from our plugin dialog more flexible, including
+supporting instalation from a url, name, or file (#2319).
```
Comment (tlambert03) [inline docs/release/release_0_4_6.md]:
```suggestion
supporting installation from anything supported by `pip`, such as a
package name, url, or local file (#2319).
```

File: docs/release/release_0_4_6.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+- Add translator stub and example (#2254)
+- Add hook implementation function name to plugin call order sorter (#2255)
+- Add `theme` parameter to `get_stylesheet` (#2263)
+- Add typing to event.py (#2269)
+- Add `canvas.background_color_override` (#2270)
+- QColoredSVGIcon class for dynamic colorized svg icons (#2279)
+- Add EventedSet class (#2280)
+- Refactor icon build process (#2285)
+- Add more tooltips (#2292)
+- Make contour thickness setable (#2296)
```
Comment (tlambert03) [inline docs/release/release_0_4_6.md]:
```suggestion
- Make contour thickness settable (#2296)
```

**Review by tlambert03 — APPROVED** [review]
looks good to me (just a couple typos)

**Comment by sofroniewn** [comment]
Thanks!! Updates made. Will merge now

**Pull request merged**